src/node_report.cc

```diff
@@ -172,13 +172,14 @@ std::string TriggerNodeReport(Isolate* isolate,
       std::cerr << " (errno: " << errno << ")" << std::endl;
       return "";
     }
+    outstream = &outfile;
 
     std::cerr << std::endl
               << "Writing Node.js report to file: " << filename << std::endl;
   }
 
   // Pass our stream about by reference, not by copying it.
-  std::ostream& out = outfile.is_open() ? outfile : *outstream;
+  std::ostream& out = *outstream;
```

Comment:
`out` is only used in one place (line 185). Could you pass `*outstream` directly there and drop `out`?

Reply:
Done PTAL.

Reply:
Thanks